### PR TITLE
Share log file

### PR DESCRIPTION
**Pull request by sdsantos**

### CODE REVIEW

File: composeApp/src/commonMain/kotlin/org/ooni/probe/shared/monitoring/CrashMonitoring.kt [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                    message = message,
+                    category = tag,
+                ),
+            )
+        }
+    }
+
+    companion object {
+        private const val SENTRY_DSN =
+            "https://9dcd83d9519844188803aa817cdcd416@o155150.ingest.sentry.io/5619989"
```
Comment (aanorbel) [inline composeApp/src/commonMain/kotlin/org/ooni/probe/shared/monitoring/CrashMonitoring.kt:91]: Should this be a configuration ?

**Review by aanorbel — APPROVED** [review]
🐳

File: composeApp/src/commonMain/kotlin/org/ooni/probe/shared/monitoring/CrashMonitoring.kt (hunk 1 above)
Comment (sdsantos) [reply composeApp/src/commonMain/kotlin/org/ooni/probe/shared/monitoring/CrashMonitoring.kt:91]: I believe it's the same for all our variations of the app. Then Sentry splits the data by application ID.

File: composeApp/src/androidMain/kotlin/org/ooni/probe/AndroidApplication.kt [hunk 2]
```diff
@@ -47,8 +54,12 @@ class AndroidApplication : Application() {
             startSingleRunInner = appWorkerManager::startSingleRun,
             configureAutoRun = appWorkerManager::configureAutoRun,
             openVpnSettings = ::openVpnSettings,
+<<<<<<< HEAD
             configureDescriptorAutoUpdate = appWorkerManager::configureDescriptorAutoUpdate,
             fetchDescriptorUpdate = appWorkerManager::fetchDescriptorUpdate,
+=======
+            shareFile = ::shareFile,
+>>>>>>> ec8eb37 (Share log file)
```
Comment (aanorbel) [inline composeApp/src/androidMain/kotlin/org/ooni/probe/AndroidApplication.kt:62]: 😕

**Pull request merged**